.golangci.yml

```diff
@@ -0,0 +1,44 @@
+run:
+  deadline: 5m
+
+linters:
+  disable-all: true
+  enable:
+    - bodyclose
+    - dogsled
+    - dupl
+    - errcheck
+    - exportloopref
+    - gocritic
+    - gocyclo
+    - gofmt
+    - goimports
+    - gosec
+    - goprintffuncname
+    - gosimple
+    - govet
+    - ineffassign
+    - misspell
+    - nakedret
+    - nolintlint
+    - revive
+    - staticcheck
+    - typecheck
+    - unconvert
+    - unused
+    - whitespace
+
+linters-settings:
+  goimports:
+    local-prefixes: sigs.k8s.io/custom-metrics-apiserver
+  misspell:
+    ignore-words:
+      - "creater" # Cf. e.g. https://pkg.go.dev/k8s.io/apimachinery@v0.24.3/pkg/runtime#ObjectCreater
+
+issues:
+  exclude-use-default: true
+  exclude-rules:
+    # We don't check duplicate code in the tests.
```

Comment:
This is odd, why do we need it?

Reply:
You're right, I should remove this exclusion.
It was for [duplicated code between `handleCustomMetrics` and `handleExternalMetrics`](https://github.com/kubernetes-sigs/custom-metrics-apiserver/blob/master/pkg/apiserver/installer/apiserver_test.go#L116-L183); but I could refactor that.